### ISSUE TITLE
[BUG]: NFL 2K5 - Stadium Parts Missing

### ISSUE THREAD

**Issue by xflamexofxhopex** [PCSX2/pcsx2#5627]
### Describe the Bug

When playing at the stadium for Jacksonville Jaguars, parts of the walls on the upper deck is missing on both sides. The texture that seems tied to this is the one attached here, from the dump. It seems the part of the texture that says Touchdown Club is the part that is missing. Not sure if that texture is applied to all of the missing parts that are not there. Here's the texture that is dumped that it uses:
![1d3418a524946523-d731f8eb8b0d3a72-00005dd3](https://user-images.githubusercontent.com/101347967/157713038-7c1c2f37-2f15-44ed-979d-b9203c67f5b9.png)
And here's the missing pieces when you view the stands (I circled where parts are missing): 
![jags1](https://user-images.githubusercontent.com/101347967/157713180-3522f25a-324a-4abb-b5de-7dfb3b4e958f.PNG)
![jags2](https://user-images.githubusercontent.com/101347967/157713204-26a6463d-b791-4b8f-8896-31f590715893.PNG)
![jags3](https://user-images.githubusercontent.com/101347967/157713224-1062e6ab-49fb-4570-aa8e-08bb43671b07.PNG)
 

### Reproduction Steps

Play any game at Altell Stadium (Jacksonville Jaguars stadium) in any weather scenario and this bug is in each weather scenario.

### Expected Behavior

Walls for stands should be fully rendered. 

### PCSX2 Revision

v1.7.2428

### Operating System

Windows 10 (64bit)

### CPU

i5-7300U

### GPU

Intel(R) HD Graphics 620

### GS Settings

Default

### Emulation Settings

Default

### Logs & Dumps

[JagsStadiumBug.zip](https://github.com/PCSX2/pcsx2/files/8225411/JagsStadiumBug.zip)

**Comment by prafullpcsx2** [comment]
Have you tried running the game on actual ps2 to confirm that those missing parts are actually visible there?

**Comment by xflamexofxhopex** [comment]
> Have you tried running the game on a real ps2 to confirm that those missing parts are actually visible there?

Yes, it shows fine in when played in PS2. The detail missing is so minute, it took a modder (me) to notice it. Everyone else emulating this game for PCSX2 for years have never pointed this out, even when this occurred pre-texture replacement. Texture replacement helps me identify what should be there.

**Comment by Mrlinkwii** [comment]
can this be check on latest mater ?
